docs/UserGuide.md

```diff
@@ -55,283 +65,419 @@ Choose a topic from the table of contents to get started on your Wishful Shrinki
+* Note: The index **must be a positive integer** e.g. 1, 2, 3......
 
 Examples:
-* `editR 2`
+* `editR 2` followed by `Enter` will insert the information of the 2nd recipe in the displayed recipe list into the command box.
+<br><br><br>
 
 ### Searching for a recipe: `searchR`
 
-Finds recipes that contains all the specified ingredients OR whose title OR tags contain any of the given keywords.
+Finds recipes in the Recipe list that contains all the specified ingredients OR whose name OR tags contain any of the given keywords.
 
-Format: `searchR i/INGREDIENT [MORE_INGREDIENT]` OR `searchR n/TITLE [MORE_TITLE]` OR `searchR t/TAG [MORE_TAG]`
+<br><br>
+<img src="images/SearchRecipeImage.png" width="550" height="300">
+<br>  The image above is what Wishful Shrinking looks like after you have searched for a recipe. In this case, the recipe is being searched by ingredients.<br><br><br>
 
-* The search is case-insensitive. e.g `salad` will match `Salad`
-* Only the recipe ingredient OR recipe title OR recipe tag is searched.
-* The order of the keywords does not matter. e.g. Ham Salad will match Salad with Ham
+Format: `searchR [i/INGREDIENT [MORE_INGREDIENT]] [n/NAME [MORE_NAME]] [t/TAG [MORE_TAG]]`
 
-Examples:
-* `searchR i/lettuce tomato` returns `salad` that has both ingredients `lettuce` and `tomato`
-* `searchR n/salad` returns `salad` and `ham salad`
-* `searchR t/healthy` returns `salad` with tag `healthy`
+* The search is case-insensitive. e.g `salad` will match `Salad`.
+* The order of the keywords does not matter. e.g. Ham Salad will match Salad with Ham.
+* Ingredients are separated by a space.
+* Note: All fields are optional, but **only one** of the fields must be present. You can only search by the recipe ingredient(s) OR recipe name OR recipe tag(s).
 
+Examples:
+* `searchR i/lettuce tomato` returns `salad` that has both ingredients `lettuce` and `tomato`.
+* `searchR n/salad` returns `salad` and `ham salad`.
+* `searchR t/healthy` returns `salad` with tag `healthy`.
+<br><br><br>
 
 ### Recommending recipes : `recommend`
 
-Shows a list of all recipes in the Recipes list that can be made with the ingredients in the user's
- fridge.
+Shows a list of all recipes in the Recipe list that can be made with the ingredients in your fridge.
+
+<br><br>
+<img src="images/RecommendImage.png" width="550" height="300">
+<br>  The image above is what Wishful Shrinking looks like after you have gotten the recommended recipes.<br><br><br>
 
 Format: `recommend`
 
+* Note: Recipes will only be recommend if you have all the ingredients in the recipe with the exact same name in your fridge.
+
 Examples:
-* `recommend` returns `salad` with ingredients `lettuce`, `onion` and `tomato` only if the user has all `lettuce`, `onion` and `tomato` in the fridge
+* `recommend` returns the recipe `salad` with ingredients `lettuce`, `onion` and `tomato` **only if** you have all `lettuce`, `onion` and `tomato` in your fridge.
```

Comment:
* `recommend` returns the recipe `salad` with ingredients `lettuce`, `onion` and `tomato` since the example user has all the ingredients: `lettuce`, `onion` and `tomato` in their fridge.